src/Jobs/AutoLiquidityManagementJob.cs

```diff
@@ -222,19 +257,34 @@ public async Task<ManageNodeLiquidityResult> ManageNodeLiquidity(Node node, Canc
                 SwapPublicationDeadlineMinutes = 30,
             };
 
-            var swapResponse = await _swapsService.CreateSwapOutAsync(
-                node,
-                SwapProvider.Loop,
-                swapRequest,
-                cancellationToken);
+            SwapResponse swapResponse;
+            
+            // Call the appropriate service based on selected provider
+            if (selectedProvider == SwapProvider.Loop)
+            {
+                swapResponse = await _swapsService.CreateSwapOutAsync(
+                    node,
+                    SwapProvider.Loop,
+                    swapRequest,
+                    cancellationToken);
+            }
+            else // SwapProvider.FortySwap
+            {
+                swapResponse = await _fortySwapService.CreateSwapOutAsync(
+                    node,
+                    swapRequest,
+                    cancellationToken);
+            }
```

Comment:
can't we call `_swapsService.CreateSwapOutAsync` and pass `SwapProvider.FortySwap` as a provider?

Reply:
100%, simplified 🏁